front/src/modules/ui/chip/components/Chip.tsx

```diff
@@ -38,8 +39,18 @@ const StyledContainer = styled.div<Partial<OwnProps>>`
   background-color: ${({ theme, variant }) =>
     variant === ChipVariant.Highlighted
       ? theme.background.transparent.light
+      : variant === ChipVariant.Rounded
+      ? theme.background.transparent.lighter
       : 'transparent'};
-  border-radius: ${({ theme }) => theme.border.radius.sm};
+  border-color: ${({ theme, variant }) =>
+    variant === ChipVariant.Rounded ? theme.border.color.medium : 'none'};
+  border-radius: ${({ theme, variant }) =>
+    variant === ChipVariant.Rounded ? '50px' : theme.border.radius.sm};
```

Comment:
'50px' => you can use theme.border.radius.rounded

Reply:
![image](https://github.com/twentyhq/twenty/assets/29349136/1dc8a7c7-b12a-443e-b2ee-cbaf88a7912d)


but it will look like this after applying rounded